### PR TITLE
main/py-pillow: rebuild against libjpeg-turbo and add check

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: main/py-pillow/APKBUILD [hunk 1]
```diff
@@ -23,6 +23,12 @@ build() {
 	python3 setup.py build
 }
 
+build() {
+	cd "$builddir"
+	python2 setup.py test
+	python3 setup.py test
```
Comment (andypost) [inline main/py-pillow/APKBUILD]: why tests running in `build()`?

Comment (J0WI) [reply main/py-pillow/APKBUILD]: Thanks for the hint, I was already wondering about the strange output.

**Comment by J0WI** [comment]
:ping_pong: @fabaff

**Comment by J0WI** [comment]
Fixed in 659180a3e5f5997a5b55774c1079c52ed097577e

**Pull request closed without merging**